packages/whale-api-client/src/whale.api.client.ts

```diff
@@ -60,6 +61,7 @@ export class WhaleApiClient {
   public readonly transactions = new Transactions(this)
   public readonly tokens = new Tokens(this)
   public readonly masternodes = new Masternodes(this)
+  public readonly block = new Blocks(this)
```

Comment:
```suggestion
  public readonly blocks = new Blocks(this)
```

On second thought, I think to call it as `blocks` instead.

Reply:
Fixed.